client/src/components/PooledMultiSelectDropdown.js

```diff
@@ -72,7 +72,7 @@ export default class PooledMultiSelectDropdown extends Component {
             customValue: e.target.value 
         })
     }
-
+    
```

Comment:
stray whitespace here, please remove

Reply:
nit: still has stray white spaces!

Reply:
bump on this nit! [this](https://packagecontrol.io/packages/Highlight%20Trailing%20Whitespace) might help